### ISSUE TITLE
[Request] Grid2StatusSpellCast implementation

### ISSUE THREAD

**Issue by Brae22** [michaelnpsp/Grid2#234]
Could this plugin for grid2 be officially implemented? https://www.curseforge.com/wow/addons/grid2statusspellcast

Despite the plugin being last updated in 2021 it still works but with bugs like the incoming spell icon getting stuck sometimes until a new spell is casted over the target or the ignored spell list not being profile-wide having to add it on every character. 

As a main healer this is a feature that in combination with _AOE & Dmg_ status option makes grid2 an insanely powerful raid frame if not the best.

**Comment by michaelnpsp** [comment]
It looks like the banzai status (statuses/combat/banzai).
There is a banzai-threat status too, to highlight when a boss targets a player (even if the player has no threat, this happens when the boss cast some spell on a player, in some special mechanics or when  a the enemy is rooted and change to the nearest melee target, for example.

**Comment by Brae22** [comment]
Hello. I've seen the update 2.8.30 adding this but I'm confused. Did this update fix the issues related with the plugin (icon stuck/list characterspecific) or did you actually implemented a new feature to banzai-threat? I don't see new options on the banzai-threat status.

**Comment by michaelnpsp** [comment]
Icon stuck issue was about the summon status not banzai statuses.
I added a blacklist option to "banzai" status not "banzai-threat" status.  But keep in mind that banzai status does not exist in classic game version only in retail. There is not reported issues for banzai or banzai-threat statuses.

**Comment by lordblackadder** [comment]
> But keep in mind that banzai status does not exist in classic game version only in retail. 

Was Banzai status disabled because of differences in CLEU between retail and classic? I enabled the status by removing the classic check, and it seems to behave as expected (in Cata at least), both functionally and without any lua errors. I've tested it both solo and in a dungeon group. I will check it in a raid as well, but I don't see how it would break there if it works with parties.
